blog/features/composite-query.md

```diff
@@ -73,7 +73,7 @@ The complete code can be found [here](https://github.com/dfinity/examples/tree/m
 An alternative implementation for Motoko can be found [here](https://github.com/dfinity/examples/tree/master/motoko/composite_query).
 
 ## Using composite queries
-To start, let's set up our development environment. Make sure you have [dfx](https://internetcomputer.org/docs/current/developer-docs/setup/install/) installed on your computer. You will need at least version 0.15.0 of dfx for composite query support. Open your terminal and follow these commands:
+To start, let's set up our development environment. Make sure you have [dfx](https://internetcomputer.org/docs/current/developer-docs/getting-started/install/) installed on your computer. You will need at least version 0.15.0 of dfx for composite query support. Open your terminal and follow these commands:
```

Comment:
I think the link here should be `/docs/current/developer-docs/getting-started/install/` (like in the file above) to ensure it works in previews as well as in production otherwise it always jumps to production where the link currently doesn't exist now.